solver.go

```diff
@@ -356,9 +357,9 @@ func (s *solver) Build(st llb.State, opts ...RequestOption) Request {
 	for _, opt := range opts {
 		opt.SetRequestOption(&r)
 	}
-	if r.download != nil {
+	if r.attachables != nil {
```

Comment:
nit: don't need this check anymore (and if kept, a `len` check would make more sense)

Reply:
yeah, not needed, left over from the updates.